packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -528,6 +531,8 @@ export default {
          */
         async submitCheckout () {
             try {
+                this.saveUserNote();
```

Comment:
Is it worth checking that the notes field has a value before calling this? Just to avoid calling it when there's no change.

Reply:
There can be a scenario when there was a user note in session storage, but a customer removed the note and hit "submit". In this case, we should update the session storage so it contains an empty string as a value. We can check if the value was changed, but, as for me, it will only make the code more complex without hardly any benefit.

Reply:
Sorry i meant if the value existed (not changed, my bad).